samcli/lib/runtimes/base.py

```diff
@@ -0,0 +1,342 @@
+import logging
+import os
+from dataclasses import dataclass, field
+from enum import Enum, unique
+from pathlib import Path
+from typing import Dict, List, Optional, Tuple, cast
+
+LOG = logging.getLogger(__name__)
+
+_init_path = Path(os.path.dirname(__file__)).parent.parent
+_templates = _init_path / "lib" / "init" / "templates"
+_lambda_images_templates = _init_path / "lib" / "init" / "image_templates"
```

Comment:
Sorry to miss this in previous review but constants are usually defined in all-uppercase

Reply:
yep I can change them in subsequent PRs